src/layout/page.js

```diff
@@ -30,6 +30,7 @@ const Page = ({
 <meta name="theme-color" content="#ffffff">
 <meta name="robots" content="index, follow">
 <meta name="author" content="Digital Transformation Agency">
+<meta name="description" content="Inclusive design, open-source code and shared insights">
```

Comment:
Where is this page being used? If it's used as a template for multiple pages then we probably need to make this more dynamic

Reply:
I don't know if it's actually being but I believe it to be the "core" page template for maybe the landing page?